packages/htmljs/visitors.js

```diff
@@ -11,16 +11,17 @@ import {
 } from './html';
 
 
-var IDENTITY = function (x) { return x; };
+const IDENTITY = function (x) {
+  return x;
+};
 
 // _assign is like _.extend or the upcoming Object.assign.
 // Copy src's own, enumerable properties onto tgt and return
 // tgt.
-var _hasOwnProperty = Object.prototype.hasOwnProperty;
-var _assign = function (tgt, src) {
-  for (var k in src) {
-    if (_hasOwnProperty.call(src, k))
-      tgt[k] = src[k];
+const _hasOwnProperty = Object.prototype.hasOwnProperty;
+const _assign = function (tgt, src) {
```

Comment:
Replace all `_assign` calls with `Object.assign` and remove this function. It's deprecated.